samples/snippets/src/main/java/com/example/spanner/BatchWriteAtLeastOnceSample.java

```diff
@@ -0,0 +1,133 @@
+
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.MutationGroup;
+import com.google.cloud.spanner.Options;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.common.collect.ImmutableList;
+import com.google.rpc.Code;
+import com.google.spanner.v1.BatchWriteResponse;
+
+public class BatchWriteAtLeastOnceSample {
+
+  /***
+   * Assume DDL for the underlying database:
+   * <p>
+   * CREATE TABLE Singers (
+   * SingerId   INT64 NOT NULL,
+   * FirstName  STRING(1024),
+   * LastName   STRING(1024),
+   * ) PRIMARY KEY (SingerId)
+   * <p>
+   * CREATE TABLE Albums (
+   * SingerId     INT64 NOT NULL,
+   * AlbumId      INT64 NOT NULL,
+   * AlbumTitle   STRING(1024),
+   * ) PRIMARY KEY (SingerId, AlbumId),
+   * INTERLEAVE IN PARENT Singers ON DELETE CASCADE
+   */
+
+  private static final MutationGroup MUTATION_GROUP1 =
+      MutationGroup.of(
+          Mutation.newInsertOrUpdateBuilder("Singers")
+              .set("SingerId")
+              .to(16)
+              .set("FirstName")
+              .to("Scarlet")
+              .set("LastName")
+              .to("Terry")
+              .build());
+  private static final MutationGroup MUTATION_GROUP2 =
+      MutationGroup.of(
+          Mutation.newInsertOrUpdateBuilder("Singers")
+              .set("SingerId")
+              .to(17)
+              .set("FirstName")
+              .to("Marc")
+              .build(),
+          Mutation.newInsertOrUpdateBuilder("Singers")
+              .set("SingerId")
+              .to(18)
+              .set("FirstName")
+              .to("Catalina")
+              .set("LastName")
+              .to("Smith")
+              .build(),
+          Mutation.newInsertOrUpdateBuilder("Albums")
+              .set("SingerId")
+              .to(17)
+              .set("AlbumId")
+              .to(1)
+              .set("AlbumTitle")
+              .to("Total Junk")
+              .build(),
+          Mutation.newInsertOrUpdateBuilder("Albums")
+              .set("SingerId")
+              .to(18)
+              .set("AlbumId")
+              .to(2)
+              .set("AlbumTitle")
+              .to("Go, Go, Go")
+              .build());
+
+  static void batchWriteAtLeastOnce() {
+    // TODO(developer): Replace these variables before running the sample.
+    final String projectId = "my-project";
+    final String instanceId = "my-instance";
+    final String databaseId = "my-database";
+    batchWriteAtLeastOnce(projectId, instanceId, databaseId);
+  }
+
+  static void batchWriteAtLeastOnce(String projectId, String instanceId, String databaseId) {
+    try (Spanner spanner =
+        SpannerOptions.newBuilder().setProjectId(projectId).setHost("https://staging-wrenchworks.sandbox.googleapis.com").build().getService()) {
```

Comment:
I think you haven't hard-coded the host intentionally. It should be removed.

Reply:
Sorry, missed removing it. Removed it now. Thanks for pointing it out.